plugins/node/opentelemetry-instrumentation-memcached/test/index.test.ts

```diff
@@ -280,7 +280,7 @@ const assertSpans = (actualSpans: any[], expectedSpans: any[]) => {
       assert.notStrictEqual(span, undefined);
       assert.notStrictEqual(expected, undefined);
       assertMatch(span.name, new RegExp(expected.op));
-      assertMatch(span.name, new RegExp(expected.key));
+      assertMatch(span.name, new RegExp('memcached'));
```

Comment:
This asserts only that the span contains the word `memcached`. It would also fail if the span name still contained the value. Do we know the possible query types so we can build a regex of possible names?

Reply:
Yes, you are right. It only asserts if it contains the word memcached. Other assertions like expected.op in the line above does operation match like memcached set, get and so on. That was the case even before this case as it was only regex matching the assertion instead of strict matching. 

Did you mean `query types` as in memcached operations like set, get? Also, I don't understand this, _building regex of possible names_, in essence, building `memcached set`, `memcached get` as individual regex instead of matching them separately is what you mean?

Reply:
It's probably fine. I just think the assertion isn't providing a lot but it really wasn't before either.